packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/price/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please use https://github.com/ovh/manager/tree/develop/packages/manager/modules/catalog-price instead

Reply:
This is done